index-de.html

```diff
@@ -0,0 +1,132 @@
+  text-align: justify;
+}
+
+a {
+  color: blue;
+}
+
+em {
+  font-weight: 400;
+}
+
+.footer {
+  padding: 40px 0;
+  background: #efefef;
+  margin: 0;
+  margin-top: 40px;
+  max-width: 100%;
+}
+
+.footer p {
+  text-align: left;
+}
+</style>
+</head>
+<body>
+<h1>Verhaltensregeln in der Community</h1>
+<p>Alle Mitglieder dieser Community sind aufgefordert, sich an folgende Verhaltensregeln zu halten. 
+Von allen Mitgliedern dieser Community wird erwartet, dass sie diese Regeln gemeinsam mit den Organisierenden bei jeder
+Veranstaltung durchsetzen.
+Wir erwarten die Mitarbeit aller Teilnehmenden der Community, um einen sicheren und angenehmen Ort für alle zu ermöglichen.</p>
+
+<h2>Brauchst du Hilfe?</h2>
+
+<p>Die Kontaktmöglichkeiten zu den Organisierenden sind auf unserer Webseite und in allen E-Mails, 
+die wir versenden, zu finden.</p>
+
+<h2>Die Kurzfassung</h2>
+<p>Wir möchten allen in unserer Community eine belästigungsfreie Erfahrung bieten, unabhängig von Geschlecht, 
+Geschlechtsidentität oder -ausdruck, Alter, sexueller Orientierung, Behinderung, Aussehen, Körperbau, Ethnizität, 
+Religionszugehörigkeit (oder Nichtvorhandensein einer solchen) oder ihrer bevorzugten Technologien. 
+Wir tolerieren keinerlei Belästigung von Mitgliedern der Community in jedweder Form. Sexualisierte Sprache und 
+Darstellungen sind bei keiner Communityveranstaltung angemessen. Dies umfasst auch Vorträge, Workshops und Partys, 
+sowie Twitter und andere Onlinemedien. Teilnehmende, die gegen diese Regeln verstoßen, müssen mit Sanktionen durch 
+die Organisierenden rechnen. Je nach deren Ermessen kann dies einen Verweises von der Veranstaltung, aller 
+zukünftigen Veranstaltungen oder aus der Community selbst beinhalten, <em>ohne Anspruch auf Rückerstattung 
+eines eventuell vorhandenen Eintrittspreises</em>.
+</p>
+
+<h2>Die längere Fassung</h2>
+
+<p>„Belästigung“ umfasst unter anderem beleidigende Aussagen über Geschlecht, Geschlechtsidentität oder -ausdruck, 
+Alter, sexueller Orientierung, Behinderung, Aussehen, Körperbau, Ethnizität, Religionszugehörigkeit, 
+bevorzugten Technologien, sexualisierende Darstellungen in öffentlichen Räumen, bewusste Einschüchterung, Stalking, 
+Verfolgung, unerwünschtes Fotografieren oder Aufnehmen, wiederholte Störung von Vorträgen und anderen Veranstaltungen, 
+unerwünschten körperlichen Kontakt und unerwünschte sexuelle Aufmerksamkeit.</p>
+
+<p>Wir erwarten, dass Teilnehmende, die gebeten werden jedwedes belästigendes Verhalten zu beenden, dem sofort nachkommen.</p>
+
+<p>Diese Verhaltensregeln gelten auch für Sponsor_innen. Insbesondere sollten Sponsor_innen darauf achten, dass sie 
+keine sexualisierte Bilder, Aktivitäten oder andere sexualisierte Werbemittel einsetzen. Teilnehmende an 
+Veranstaltungen und Gastgebende, inklusive Sponsor_innen, Standpersonal und Freiwillige sollten keine sexualisierte 
+Kleidung, Uniformen oder Kostüme tragen, oder sonstwie ein sexualisiertes Umfeld gestalten.</p>
```

Comment:
"sonstwie" ist alte Rechtschreibung, neu waere "sonst wie". Wuerde aber eher zu "oder in anderer Weise" tendieren